app/models/user.rb

```diff
@@ -54,22 +54,53 @@ def administrated_projects
     scope
   end
 
-  def self.to_csv
-    @users = User.order(:id)
+  def self.to_csv(
+    inherited: false, deleted: false, project_id: nil, user_id: nil,
+    datetime: (Time.now.strftime "%Y%m%d_%H%M")
+  )
+    inherited = true if project_id || user_id
```

Comment:
Had this since when reporting on a specific project it would be a bit off to see anywhere between less than 100% of the users.  
Same with reporting on a specific user to see less than 100% of the projects, since most of their privileges are associated with their system permissions.